EIPS/eip-1024.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 1024
+title: Add web3.eth.encrypt and web3.eth.decrypt functions
```

Comment:
Would make sense including "to JSON-RPC" here.

Reply:
```suggestion
title: Add web3.eth.encrypt and web3.eth.decrypt functions to JSON-RPC
```